spec/mailers/jobseekers/account_mailer_spec.rb

```diff
@@ -14,6 +14,17 @@
     }
   end
 
+  describe "#account_closed" do
+    let(:mail) { described_class.account_closed(jobseeker) }
+    let(:notify_template) { NOTIFY_JOBSEEKER_ACCOUNT_CLOSED_TEMPLATE }
+
+    it "sends an `account_closed` email" do
```

Comment:
I dunno if this is a valid thing to bring up, but is this test testing that the email is sent, or that the email is constructed correctly?

Reply:
a test cannot send an email for real. it tests that it is constructed correctly

Reply:
That's what I mean. So perhaps "sends an `account_closed` email" as the name of the spec isn't the best? I dunno, just a small suggestion!